### PR TITLE
Make Application.add public again

### PR DESCRIPTION
**Pull request by ketzusaka**

### CODE REVIEW

File: Sources/Vapor/Routing/Application+Route.swift [hunk 1]
```diff
@@ -81,7 +81,7 @@ extension Application {
         }
     }
 
-    final func add(method: Request.Method, path: String, handler: Route.Handler) {
+    public final func add(method: Request.Method, path: String, handler: Route.Handler) {
```
Comment (LoganWright) [inline Sources/Vapor/Routing/Application+Route.swift]: @ketzusaka Do you want to add some documentation to this?

Comment (ketzusaka) [inline Sources/Vapor/Routing/Application+Route.swift]: yeah x.x First commit from me since Hound, sorry about that

Comment (tannernelson) [inline Sources/Vapor/Routing/Application+Route.swift]: Make sure to format it like the other public docs please :)

```swift
/**
    Description blah blah blah.
    
    - parameter: blah blah
*/
func asdf() {}
```

File: Sources/Vapor/Routing/Application+Route.swift [hunk 2]
```diff
@@ -81,7 +81,16 @@ extension Application {
...
+    /**
+     Adds a route handler for an HTTP request using a given HTTP verb at a given
+     path. The provided handler will be ran whenever the path is requested with 
+     the given method.
+     
+     - parameter    method:     The `Request.Method` that the handler should be executed for.
+     - parameter    path:       The HTTP path that handler can run at.
+     - parameter    handler:    The code to process the request with.
+    */
+    public final func add(method: Request.Method, path: String, handler: Route.Handler) {
```
Comment (tannernelson) [inline Sources/Vapor/Routing/Application+Route.swift]: Make sure the docs are tabbed over. They look a little weird lined up like that.

```swift
/**
    Hello world.
*/```

**Comment by ketzusaka** [comment]
@tannernelson can we get this merged in, i'd like it to be in the 04-12 tag

**Comment by LoganWright** [comment]
@ketzusaka Once I get 04-12 tests passing, I'll pop this commit on top 👍

**Comment by LoganWright** [comment]
Closing this for #186 I popped these commits onto that branch

**Pull request closed without merging**